modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/AbstractCSharpCodegen.java

```diff
@@ -603,6 +603,8 @@ public Map<String, ModelsMap> postProcessAllModels(Map<String, ModelsMap> objs)
                         property.name = patchPropertyName(model, camelize(property.baseType));
                         property.isNullable = true;
                         patchPropertyVendorExtensions(property);
+                        String baseName = model.name.substring(model.name.lastIndexOf('_') + 1);
+                        property.vendorExtensions.put("x-base-name", baseName);
```

Comment:
what about using just 1 line without the baseName (i.e. the put value is `model.name.substring(model.name.lastIndexOf('_') + 1)` ? as baseName isn't used any where else?

Reply:
Done, though I don't think I should need to extract the base name, it should be made available in someway when the property is created.